js/app/modules/card/defaultFamily.js

```diff
@@ -192,21 +291,33 @@ const Thumb = new Module.Class({
     vfunc_size_allocate: function (alloc) {
         this._layout.clear_constraints();
```

Comment:
I agree with @rmacqueen; I'm looking at implementing some additional API in Emeus to have "constraint groups" to have semantic aggregates of constraints, and especially to ensure that switching between different layouts is not slowed down by the removal/addition of all constraints.